conans/client/loader.py

```diff
@@ -261,7 +261,10 @@ def _parse_conanfile(conan_file_path):
             module = sys.modules[added]
             if module:
                 try:
-                    folder = os.path.dirname(module.__file__)
+                    try:
+                        folder = os.path.dirname(module.__file__)
+                    except (AttributeError, TypeError):  # Namespace packages py3 module.__file__
+                        folder = module.__path__._path[0]
                 except AttributeError:  # some module doesn't have __file__
```

Comment:
So, this except is for the `module.__path__._path[0]`? The comment is not very clear.